doc/running/caching.rst

```diff
@@ -0,0 +1,52 @@
+Caching in mirgecom
+===================
+
+
+OpenCL Kernels
+--------------
+
+OpenCL kernels are cached on hard disks on multiple levels in a mirgecom
+execution. This has the advantage of reducing the compilation time of kernels
+when running the same driver multiple times, but can lead to inconsistent
+performance results, especially when comparing multiple executions against
+each other, as subsequent executions can appear faster than earlier ones due
+to caching effects.
+
+
+The following list discusses mirgecom-related packages that use caching.
+
+Loopy
++++++
+
+Loopy stores generated PyOpenCL kernels in ``$XDG_CACHE_HOME`` by default. You can
+export ``LOOPY_NO_CACHE=1`` to disable caching. See
+`here <https://github.com/inducer/loopy/blob/e21e8f85d289abbca27ac6abfd71874155fa49da/loopy/__init__.py#L402-L406>`__
+for details.
```

Comment:
You could explain the directory names a bit more precisely.

Reply:
Added.